### PR TITLE
Update existing user

### PR DESCRIPTION
**Pull request by munteanumarco**

### CODE REVIEW

**Review by gheorghe-sava — APPROVED** [review]
Looks good!

File: collaborative_practice.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const { lstatSync } = require('fs');
```
Comment (gheorghe-sava) [inline collaborative_practice.js]: Why was this required?

Comment (munteanumarco) [reply collaborative_practice.js]: yep, it slipped, I removed it when I resolved the merge conflicts

**Review by gheorghe-sava — APPROVED** [review]

**Review by bObXm — APPROVED** [review]
